ipi_tests/unit_tests/utils/io/backends/test__init__.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 import numpy.testing as npt
 
-import ipi_tests.unit_tests.common.xyz_generator as xyz_gen
+from ....common import xyz_generator as xyz_gen
```

Comment:
If we restructure any of the folder tree here, all of this will have to be addressed, am I right? just a question, but still...

Reply:
That's true. There isn't that many of them, though, and I assume the restructuring wouldn't be done too often, so hopefully it shouldn't be an issue?